lib/cb/requests/job_search/get.rb

```diff
@@ -18,6 +18,11 @@ def headers
             'Accept' => 'applicatin/json;version=3.0'
           }
         end
+        
+        def query
+          args.except(:three_scale_token)
```

Comment:
It looks like, from other request objects, the query arguments typically have camel case rather than snake case. Is that an expected issue here?